### ISSUE TITLE
Loss is extreamly small from the very beginning.

### ISSUE THREAD

**Issue by kawori** [yu4u/noise2noise#8]
I found that when training with an image with different noises, the MSE loss is very small from the beginning and then underflow.

**Comment by kawori** [comment]
What causes this issue is that I normalized the images to the range [0, 1].

**Comment by yu4u** [comment]
Is it solved?

**Comment by kawori** [comment]
Solved.
I'm using UNet for training, but it's hard to converge. It's the same for RedNet.

**Comment by kawori** [comment]
When using UNet, the loss decresed fast at first, but then decreased slowly and increased to INF suddenly.

**Comment by kawori** [comment]
I've create a repo here. You can review my codes though it's PyTorch implementation.

**Comment by yu4u** [comment]
I added UNet to this repository,  and trained it.
It works and seems to be better than SRResNet in case of Gaussian noise.
I think the problem is not due to the UNet model.

**Comment by kawori** [comment]
My UNet model has no BN and dropout layer. Its detailed parameters are from Noise2Noise paper's appendix.

**Comment by yu4u** [comment]
Even if the model is trained with no BN and dropout, it works in my implementation.
This can be done by calling `get_unet_model` with `dropout=0`.

https://github.com/yu4u/noise2noise/blob/master/model.py#L60-L61

**Comment by rivergold** [comment]
I meet the same situation as @kawori. When I use u-net, the loss is hard to converge, It decresed fast at first and then decreased slowly and increased to INF suddenly.

**Comment by yu4u** [comment]
@rivergold hmm... Was it happened on your implementation? On noisy targets' training and/or clean targets' training?

**Comment by rivergold** [comment]
@yu4u Thanks for your reply. When I use your keras code training on noisy target to remove text in image, the final loss is about 20. And I rewrite the code with TensorFlow train with same parameter, the loss decresed fast at first and then decreased slowly and increased to INF suddenly.

**Comment by yu4u** [comment]
Thank you for your information.
I'm afraid that I could not give useful comment on why TensorFlow implementation does not work while Keras implementation does...

**Comment by Shakarim94** [comment]
Did you guys fix the issue of the INF loss? I tried to make the training as close to the original paper as possible. Using the Imagenet validation set and the same U-Net as in paper. But still getting INF loss in the 2nd epoch

**Comment by yu4u** [comment]
Hi, @Shakarim94 

> Did you do any pre-processing such as image normalization, scaling or mean subtraction to avoid this?

No, the input input and target images are np.uint8 array in [0, 255].
